wrappers/javascript/aries-askar-nodejs/src/index.ts

```diff
@@ -1 +1,8 @@
-export { NodeJSAriesAskar } from './NodeJSAriesAskar'
+import { registerAriesAskar } from '@hyperledger/aries-askar-shared'
+
+import { NodeJSAriesAskar } from './NodeJSAriesAskar'
+
+export const ariesAskarNodeJS = new NodeJSAriesAskar()
+registerAriesAskar({ askar: ariesAskarNodeJS })
```

Comment:
This method of registering based on a simple import before everything else remembers me the `reflect-metadata` that we need to import before using class-transformer. Not sure if it's completely intuitive, but certainly this method or registering is simpler than the previous one, and as long as it's properly documented (as I've seen in the README) it's fine to me.